Makefile

```diff
@@ -381,15 +381,18 @@ watcher: export CATALOG_IMG=${CATALOG_IMAGE}
 watcher: ## Install watcher operator via olm
 	bash ci/olm.sh
 	oc apply -f ci/olm.yaml
+	timeout 300s bash -c "while ! (oc get csv -n  openshift-operators -l  operators.coreos.com/cluster-observability-operator.openshift-operators -o jsonpath='{.items[*].status.phase}' | grep Succeeded); do sleep 10; done"
 	timeout 300s bash -c "while ! (oc get csv -n openstack-operators -l operators.coreos.com/watcher-operator.openstack-operators -o jsonpath='{.items[*].status.phase}' | grep Succeeded); do sleep 1; done"
 
 .PHONY: watcher_deploy
 watcher_deploy: ## Deploy watcher service
+	oc apply -f config/samples/watcher_requirements.yaml
```

Comment:
we proably shoudl be waiting in watcher_deploy and watcher_deploy_cleanup too

again like we do for kuttle

https://github.com/openstack-k8s-operators/nova-operator/blob/main/Makefile#L400-L417

Reply:
yep, I'm adding a wait for watcher CR after creating it.